app/src/main/java/com/beemdevelopment/aegis/otp/YandexInfo.java

```diff
@@ -0,0 +1,72 @@
+package com.beemdevelopment.aegis.otp;
+
+import com.beemdevelopment.aegis.crypto.otp.YAOTP;
+import com.beemdevelopment.aegis.encoding.Base32;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.nio.charset.StandardCharsets;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.util.Locale;
+
+public class YandexInfo extends TotpInfo {
```

Comment:
This class is missing an override for ``equals``, which is used to compare entries. Something like:

```java
@Override
public boolean equals(Object o) {
    if (!(o instanceof YandexInfo)) {
        return false;
    }

    YandexInfo info = (YandexInfo) o;
    return super.equals(o) && getPin() == info.getPin();
}
```

Reply:
Added, thanks for noticing!